source/defender.c

```diff
@@ -265,18 +266,19 @@ static void writeFormatAndSuffix( char * pBuffer,
                     DEFENDER_API_ACCEPTED_SUFFIX_LENGTH );
             break;
 
+        /* The default is here just to silence compiler warnings in a way which
+         * does not bring coverage down. The assert at the beginning of this
+         * function ensures that the only api value hitting this case can be
+         * DefenderCborReportRejected. */
```

Comment:
@markrtuttle I wonder if there's a way to enforce that this comment remains true in CBMC?

For example, could we move the code up to the `case` block and put `assert(0)` in the `default` block? Or one of CBMC's macros like `__CPROVER_assert(0)`?

Reply:
I think adding a
```
    default:
        assert(0);
```
is a reasonable idea.